kits/agent-reliability-lab/prompts/agent-reliability-audit_instructor-llmnode-736_user_1.md

```
@@ -0,0 +1,7 @@
+ORIGINAL SYSTEM PROMPT:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)